### ISSUE TITLE
Using -i and -C causes segmentation fault

### ISSUE THREAD

**Issue by ehwilliamson** [kstrauser/pgdbf#39]
If CREATE is suppressed, field names are never created:
https://github.com/kstrauser/pgdbf/blob/4e847759b5eee96da9dc3d2604e0b555ddd57ccb/src/pgdbf.c#L476-L477

Those field names are used in the ignore code: https://github.com/kstrauser/pgdbf/blob/4e847759b5eee96da9dc3d2604e0b555ddd57ccb/src/pgdbf.c#L538

Using -i with -C (or -u) cause a segfault when strcmp tries to read a nonexistent fieldname.

**Comment by rubenchila** [comment]
Have you ever solved this issue?

**Comment by ehwilliamson** [comment]
yeah I just forced that `if` in line 476 to be true by changing it to `if(1)`. That fixes it for me.
